app/views/mountain_view/styleguide/_main_page_content.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="mv-main__header">
+  <h1><%= t('.mountain_view.layout.styleguide_title') %></h1>
+</div>
```

Comment:
This is now extracted inside the conditional but also added to the `else` part of it. Any reason to do that rather than keeping it outside the conditional as it was before?

Reply:
I suppose we could keep it out, but that mandates it be present on the page.  My thought was that when they have not created components, the main page must display the instructions for creating the components, and at that point we get to control the h1's placement and display.  But once they have components, and the partial is in use and can be replaced, the h1 should be within the partial so that the programmer has control over where and whether the h1 exists on the style guide main page.

Reply:
We could have different titles based on whether it is the initial instructions or the partial... I don't know if that would be useful or not.

Reply:
OK, fair enough!